examples/stm32/xively_demo_f4xx_nucleo_wifi/SW4STM32/STM32F401RE-Nucleo/STM32F4xx-Nucleo-Server_Socket/.project

```diff
@@ -0,0 +1,238 @@
+		</link>
+		<link>
+			<name>Drivers/STM32F4xx_HAL_Driver/stm32f4xx_hal_flash_ex.c</name>
+			<type>1</type>
+			<locationURI>STM_CUBE_PATH/Drivers/STM32F4xx_HAL_Driver/Src/stm32f4xx_hal_flash_ex.c</locationURI>
+		</link>
+		<link>
+			<name>Drivers/STM32F4xx_HAL_Driver/stm32f4xx_hal_gpio.c</name>
+			<type>1</type>
+			<locationURI>STM_CUBE_PATH/Drivers/STM32F4xx_HAL_Driver/Src/stm32f4xx_hal_gpio.c</locationURI>
+		</link>
+		<link>
+			<name>Drivers/STM32F4xx_HAL_Driver/stm32f4xx_hal_rcc.c</name>
+			<type>1</type>
+			<locationURI>STM_CUBE_PATH/Drivers/STM32F4xx_HAL_Driver/Src/stm32f4xx_hal_rcc.c</locationURI>
+		</link>
+		<link>
+			<name>Drivers/STM32F4xx_HAL_Driver/stm32f4xx_hal_rcc_ex.c</name>
+			<type>1</type>
+			<locationURI>STM_CUBE_PATH/Drivers/STM32F4xx_HAL_Driver/Src/stm32f4xx_hal_rcc_ex.c</locationURI>
+		</link>
+		<link>
+			<name>Drivers/STM32F4xx_HAL_Driver/stm32f4xx_hal_tim.c</name>
+			<type>1</type>
+			<locationURI>STM_CUBE_PATH/Drivers/STM32F4xx_HAL_Driver/Src/stm32f4xx_hal_tim.c</locationURI>
+		</link>
+		<link>
+			<name>Drivers/STM32F4xx_HAL_Driver/stm32f4xx_hal_tim_ex.c</name>
+			<type>1</type>
+			<locationURI>STM_CUBE_PATH/Drivers/STM32F4xx_HAL_Driver/Src/stm32f4xx_hal_tim_ex.c</locationURI>
+		</link>
+		<link>
+			<name>Drivers/STM32F4xx_HAL_Driver/stm32f4xx_hal_uart.c</name>
+			<type>1</type>
+			<locationURI>STM_CUBE_PATH/Drivers/STM32F4xx_HAL_Driver/Src/stm32f4xx_hal_uart.c</locationURI>
+		</link>
+		<link>
+			<name>Drivers/STM32F4xx_HAL_Driver/stm32f4xx_hal_usart.c</name>
+			<type>1</type>
+			<locationURI>STM_CUBE_PATH/Drivers/STM32F4xx_HAL_Driver/Src/stm32f4xx_hal_usart.c</locationURI>
+		</link>
+		<link>
+			<name>Example/SW4STM32/startup_stm32f401xe.s</name>
+			<type>1</type>
+			<locationURI>$%7BPARENT-1-PROJECT_LOC%7D/startup_stm32f401xe.s</locationURI>
+		</link>
+		<link>
+			<name>Example/User/console.c</name>
+			<type>1</type>
+			<locationURI>$%7BPARENT-3-PROJECT_LOC%7D/Src/console.c</locationURI>
+		</link>
+		<link>
+			<name>Example/User/main.c</name>
+			<type>1</type>
+			<locationURI>$%7BPARENT-3-PROJECT_LOC%7D/Src/main.c</locationURI>
+		</link>
+		<link>
+			<name>Example/User/stm32_xx_hal_msp.c</name>
+			<type>1</type>
+			<locationURI>$%7BPARENT-3-PROJECT_LOC%7D/Src/stm32_xx_hal_msp.c</locationURI>
+		</link>
+		<link>
+			<name>Example/User/stm32_xx_it.c</name>
+			<type>1</type>
+			<locationURI>$%7BPARENT-3-PROJECT_LOC%7D/Src/stm32_xx_it.c</locationURI>
+		</link>
+		<link>
+			<name>Example/User/syscalls.c</name>
+			<type>1</type>
+			<locationURI>$%7BPARENT-1-PROJECT_LOC%7D/syscalls.c</locationURI>
+		</link>
+		<link>
+			<name>Middlewares/ST/STM32_SPWF01SA</name>
+			<type>2</type>
+			<locationURI>virtual:/virtual</locationURI>
+		</link>
+		<link>
+			<name>Drivers/BSP/STM32F4xx-Nucleo/stm32f4xx_nucleo.c</name>
+			<type>1</type>
+			<locationURI>STM_CUBE_PATH/Drivers/BSP/STM32F4xx-Nucleo/stm32f4xx_nucleo.c</locationURI>

```

Comment:
STM_CUBE_PATH and STM_CUBE_WIFI_PATH are the same. Please leave only either, thx.